Warn when response body is larger than 5mb (#26831)
vercel/next.js: errors/api-routes-body-size-limit.md

> # API Routes body size limited to 5mb

Reviewer: ```suggestion
# API Routes Body Size Limited to 5mb
```

We usually use [title case](https://title.sh) here

#### Why This Error Occurred

> API routes are meant to respond quickly and are not intended to support responding with large globs of data. The size limit for responses is set to 5mb.

Reviewer: ```suggestion
API routes are meant to respond quickly and are not intended to support responding with large amounts of data. The maximum size of responses is 5mb.
```

#### Possible Ways to Fix It

> Limit your api route responses to less than 5mb. If you need to support sending large files to the client, you should consider using a dedicated media host for those assets. See link below for suggestions.

Reviewer: ```suggestion
Limit your API Route responses to less than 5mb. If you need to support sending large files to the client, you should consider using a dedicated media host for those assets. See link below for suggestions.
```

### Useful Links

> [Bypass 5mb body size limit](https://vercel.com/support/articles/how-to-bypass-vercel-5mb-body-size-limit-serverless-functions?query=5mb#request-directly-from-the-source)

Reviewer 2: ```suggestion
[Tips to avoid the 5mb limit](https://vercel.com/support/articles/how-to-bypass-vercel-5mb-body-size-limit-serverless-functions)
```

Might be good to reword the link a bit and it looks like the query from searching for the doc got left on it so removed that since it isn't needed

> [Bypass 5mb body size limit](https://vercel.com/support/articles/how-to-bypass-vercel-5mb-body-size-limit-serverless-functions?query=5mb#request-directly-from-the-source)

Reviewer: ```suggestion
[Bypass 5mb body size limit](https://vercel.com/support/articles/how-to-bypass-vercel-5mb-body-size-limit-serverless-functions)
```

Probably link to the page instead of a specific section
